main.tf

```diff
@@ -10,4 +10,11 @@ resource "aws_lb" "test" {
   tags = {
     Environment = "production"
   }
+
+  access_logs {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure cross-zone load balancing is enabled for AWS LB (Load Balancer)**
 
**Vulnerability**
AWS Load Balancer has cross-zone load balancing disabled. This will prevent movement of traffic to registered targets outside the availability zone.
 
**Remediation**
Cross-zone load balancing should be enabled. When cross-zone load balancing is enabled, each load balancer node distributes traffic across the registered targets in all enabled Availability Zones.

source: accurics-stagetwo